### DISCUSSION TITLE
How can I get the Intellisense of the installed nodemodules, and written scripts?

### DISCUSSION THREAD

**Discussion by crimson206** [microsoft/monaco-editor#4517] [Q&A]
I want to make a playground whose target is the whole environment. Is it possible?

```
devDependency
@fluent-ui
more modules...
```
```
import MyComponent './MyFile'
import Button from '@fluent-ui'
```

When I write the second code block in the editor, I want the Intellisense of them automatically.

**Comment by mnahkies** [comment]
You might find https://github.com/lukasbach/monaco-editor-auto-typings/ helpful - it can automatically download `.d.ts` files and add models for them to the editor. I'm using it like:
```
import {
  AutoTypings,
  LocalStorageCache,
} from "monaco-editor-auto-typings/custom-editor"

AutoTypings.create(editor, {
        monaco,
        fileRootPath: "file:///",
        sourceCache: new LocalStorageCache(),
})
```
